tests/get_or_create/tests.py

```diff
@@ -535,6 +535,69 @@ def lock_wait():
         self.assertGreater(after_update - before_start, timedelta(seconds=0.5))
         self.assertEqual(updated_person.last_name, 'NotLennon')
 
+    @skipUnlessDBFeature('has_select_for_update')
+    @skipUnlessDBFeature('supports_transactions')
+    def test_creation_in_transaction(self):
```

Comment:
Note that this test was based on the existing `test_updates_in_transaction` test and so skips if the DB lock takes too long (as apparently Oracle can do this sometimes).